MekHQ/build.gradle

```diff
@@ -125,7 +127,7 @@ task mmJarLibDir (type: Jar, dependsOn: compileMM) {
             .findAll { it.name.endsWith(".jar") && !it.name.toLowerCase().startsWith("megamek") }
             .collect { it.name }.join(' ')
     }
-    ext.jarLocation = "${buildDir}/libs/${archiveName}"
+    ext.jarLocation = "${buildDir}/libs/${archiveFileName}"
     inputs.dir "${mmDir}/build/classes/java/main"
```

Comment:
Does these paths need `megamek` added too?

Reply:
No. Those that were added to the compileMM task were to find the class and resource files to place into the jar in the build directory of the megamek subdirectory rather than the top level. But once built, the jar is placed in the libs subdirectory of the current build directory, which is where it looks to find it in this task.